src/js/view/controls/nextuptooltip.js

```diff
@@ -100,7 +100,7 @@ export default class NextUpTooltip {
         setTimeout(() => {
```

Comment:
@radium-v @hghazzi is there a better explanation for why this is in a timeout? What animation (CSS class) for which item (does next up change mid playlist)?

Reply:
@robwalch the animation is from the toggling of `jw-nextup-container-visible` that occurs on line 83. It's to ensure that the contents of the next up container are updated after the class is removed at the end of the playlist item